PowerEditor/src/NppNotification.cpp

```diff
@@ -59,6 +59,63 @@ BOOL Notepad_plus::notify(SCNotification *notification)
 			{
 				// for the backup system
 				_pEditView->getCurrentBuffer()->setModifiedStatus(true);
+
+				// auto make temporary name for untitled documents
+				Buffer* buffer = notifyView->getCurrentBuffer();
+				const NewDocDefaultSettings& ndds = NppParameters::getInstance().getNppGUI().getNewDocDefaultSettings();
+				intptr_t curLineIndex = _pEditView->execute(SCI_LINEFROMPOSITION, notification->position);
+				if (curLineIndex == 0 && ndds._autoMakeTempName && buffer->isUntitled() && !buffer->isRenamed())
+				{
+					// make a temporary file name from first line
+					wstring tempFileName = _pEditView->getLine(0);

```

Comment:
use `content1stLineTabName` instead of `tempFileName`, and change the comment properly.